ui.frontend/src/view/FormFieldBase.js

```diff
@@ -445,7 +448,7 @@ class FormFieldBase extends FormField {
         if (this.errorDiv) {
             this.errorDiv.innerHTML = state.validationMessage;
             if (state.validity.valid === false) {
-                this.#triggerEventOnGuideBridge(this.ELEMENT_ERROR_SHOWN);
+                this.#triggerEventOnGuideBridge(this.ELEMENT_ERROR_SHOWN, {'validationMessage' : state.validationMessage});
```

Comment:
`validationMessage` could have PII data, hence we would instead send the `validationType` in the beacon.